### PR TITLE
Solution and test for issue #7

### PR DESCRIPTION
**Pull request by yjlim5**

### CODE REVIEW

File: test/7.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const expect = require('chai').expect;
-// require your getLongestInc function
+const arrayOp = require('../solutions/7');
+const getLongestInc = arrayOp.getLongestInc;
 
-/* describe('getLongestInc', () => {
+describe('getLongestInc', () => {
     it('should return the number of longest increasing numbers in an array', () => {
       let arr = [2,3];
```
Comment (songz) [inline test/7.js]: test cases needs to be grouped and separated. for example:
it should return the correct result when middle number is decreasing

File: solutions/7.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  let max = 0;
+  indicesArray.forEach( (index) => {
+    if (arr[index] > max) {
+      max = arr[index];
+    }
+  })
+  return max;
+}
+
+const getMax = (arr) => {
```
Comment (songz) [inline solutions/7.js]: no need to create a new function, use es6 Max method: `Math.max(...arr);`

File: solutions/7.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  let indices = [];
+  for (let i = startIndex; i > 0; i--) {
+    if (arr[i-1] < arr[startIndex]) {
+      indices.push(i-1);
+    }
+  };
+  return indices;
+}
+
+const getMaxAmongIndices = (arr, indicesArray) => {
```
Comment (songz) [inline solutions/7.js]: Think about this:

```
var arr = [1,2,3];
var max = arr.reduce((a, b) => {
    return Math.max(arr[a], arr[b]);
});
```

File: test/7.js (hunk 1 above)
Comment (yjlim5) [inline test/7.js]: done.

File: solutions/7.js (hunk 3 above)
Comment (yjlim5) [inline solutions/7.js]: `  return indicesArray.reduce( (acc, b) => {
    return Math.max(acc, arr[b]);
  },0);`

File: solutions/7.js (hunk 2 above)
Comment (yjlim5) [inline solutions/7.js]: done

File: solutions/7.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  let indices = [];
+  for (let i = startIndex; i > 0; i--) {
+    if (arr[i-1] < arr[startIndex]) {
+      indices.push(i-1);
+    }
+  };
+  return indices;
+};
+
+const getMaxAmongIndices = (arr, indicesArray) => {
```
Comment (songz) [inline solutions/7.js]: combine this with `getIndicesOfSmallerVals`

File: test/7.js [hunk 5]
```diff
@@ -1,28 +1,45 @@
...
-// require your getLongestInc function
+const solution = require('../solutions/7').solution;
+// solution = require('../yourSolution').solution;
 
-/* describe('getLongestInc', () => {
-    it('should return the number of longest increasing numbers in an array', () => {
-      let arr = [2,3];
-      expect(getLongestInc(arr)).to.equal(2);
+describe('solution should return the number of longest increasing numbers in', () => {
+  let arr = [];
```
Comment (songz) [inline test/7.js]: seems like you don't need arr

**Pull request merged**